project_info/bin/release-bitbucket.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+import click
+
+from project_info.git_utils import get_current_branch_name
+from project_info.bitbucket_utils import create_merge_release_pull_request as create_merge_release_pull_request_func
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command()
+@click.option('--username', help='username', type=str, required=True)
+@click.option('--password', help='password', type=str, required=True)
+@click.option('--source_branch_name', help='source git branch name', type=str)
+@click.option('--destination_branch_name', help='destination git branch name', type=str, required=True, default='next')
+@click.option('--repository_name', help='git repository name', type=str, required=True)
+def create_merge_release_pull_request(username, password, source_branch_name, destination_branch_name, repository_name):
```

Comment:
helper for bitbucket. Maybe we can use similar in the standard deployment. If find way how get title from jira issue :)